components/create/CreateNft.vue

```diff
@@ -354,6 +370,23 @@ const deposit = computed(() =>
   (Number(totalItemDeposit.value) * form.copies).toFixed(4),
 )
 
+// usd value
+
+// when left undefined urlPrefix will be used
+const tokenType = computed(() =>
+  isBasilisk.value ? chainSymbol.value.toLowerCase() : undefined,
+)
```

Comment:
What's for?

Reply:
Chain symbol is the fees asset chosen by user.
This is only relevant on Basilisk chain

Can be deleted when we deprecate bsx

Reply:
hmmmm